### ISSUE TITLE
Custom tls certificate files name

### ISSUE THREAD

**Issue by ctinmota** [fabric8io/docker-maven-plugin#792]
Hi,

Is there any possibility to specify custom tls certificate files name?
When using -Ddocker.certPath=%docker.cert.path% it looks for standard names ca.pem, cert.pem and key.pem. 
I would like to know if there is a way to specify each file separately using custom names for them?

Thank you,
Constantin Mota

**Comment by rhuss** [comment]
No, the plugin expects the certs with the names as defined in https://docs.docker.com/engine/security/https/#client-modes

**Comment by ctinmota** [comment]
Hi,

Thank you for your fast response! 
Is there any chance in future to have support for this requirement?

**Comment by rhuss** [comment]
Could you please briefly describe your use case and why do you need custom certificate names and why the defaults are not good enough for you ?

**Comment by ctinmota** [comment]
We are using team city for building our releases, using multiple agents, and is possible to use different docker hosts, depending on projects, with different certificates and it will be nice to store all these certificates in some shared location for all agents. 
For the moment we will organize them in different directories.
